recipe-server/client/control_new/components/extensions/CreateExtensionPage.js

```diff
@@ -0,0 +1,57 @@
+import { message } from 'antd';
+import autobind from 'autobind-decorator';
+import React, { PropTypes as pt } from 'react';
+import { connect } from 'react-redux';
+import { push } from 'redux-little-router';
+
+import ExtensionForm from 'control_new/components/extensions/ExtensionForm';
+import { createExtension } from 'control_new/state/extensions/actions';
+
+
+@autobind
+export class _CreateExtensionPage extends React.Component {
+  static propTypes = {
+    createExtension: pt.func.isRequired,
+    push: pt.func.isRequired,
+  }
+
+  constructor(props) {
```

Comment:
We _should_ be able to just add `state` to the class, instead of using the constructor, e.g.:

```
class Whatever extends React.Component {
  static propTypes = {...};
  state = { formErrors: undefined, };

  // ..component functions..
}
```